src/components/event/Create.tsx

```diff
@@ -134,7 +134,7 @@ export const CreateEvent: React.FC<ICreateEventProps> = props => {
 
   const handleChangeStartDate = (e: any) => {
     const startDateString = e.target.value;
-    const startDate = new Date(`${startDateString}T00:00:00`);
+    const startDate = +new Date(`${startDateString}T00:00:00`);
```

Comment:
why the `+` sign?

Reply:
It coerces the date into a number same as  "dateObject.getTime()"

Reply:
Too tricky... favor `dateObject.getTime()`